uniswap/uniswap.py

```diff
@@ -730,10 +740,12 @@ def _token_to_token_swap_input(
                 function = token_funcs.tokenToTokenTransferInput(*func_params)
             return self._build_and_send_tx(function)
         elif self.version == 2:
+            if not route:
+                route = [input_token, self.get_weth_address(), output_token]
```

Comment:
WETH in the middle is an assumption, it doesn't have to be like this. Imagine swapping stable for stable. Probably the most correct approach is just `[input_token, output token]`

Reply:
I can't really say if it's correct or not as i'm not terribly familiar with this stuff but I didn't want to make any breaking changes